framework/video_recorder.py

```diff
@@ -54,6 +54,6 @@ def record_trajectory(self, trajectory: Trajectory, run_name: str, fps=30):
 
             # Save the rendered image to the video file
             writer.write(env.render())
-
+        print("video_done")
```

Comment:
I would remove this to avoid the console being flooded. Maybe we should consider adding a better logger for debug messages like this one in the future.